src/AnalysisServices/AnalysisServices.Dataplane/Models/AsAzureEndpoints.cs

```diff
@@ -12,31 +12,19 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using System;
-using System.Collections;
-using System.Collections.Generic;
-
-namespace Microsoft.Azure.Commands.AnalysisServices.Dataplane
+namespace Microsoft.Azure.Commands.AnalysisServices.Dataplane.Models
 {
-    [Serializable]
-    public class AsAzureEnvironment
+    /// <summary>
+    /// Class to hold the endpoint format strings for dataplane cmdlets.
+    /// </summary>
+    class AsAzureEndpoints
     {
-        public AsAzureEnvironment(string Name)
-        {
-            this.Name = Name;
-            this.Endpoints = new Hashtable();
-        }
-
-        public string Name { get; set; }
+        public const string RestartEndpointPathFormat = "/webapi/servers/{0}/restart?api-version=2016-10-01";
```

Comment:
did we hard code api version previously?

Reply:
Yes
https://github.com/Azure/azure-powershell/blob/14c9bf81a4b76bece8b652db0622a6cddaea97b5/src/AnalysisServices/AnalysisServices.Dataplane/Models/AsAzureClientSession.cs#L29